spec/classes/init_spec.rb

```diff
@@ -960,6 +966,26 @@
       end
     end
   end
+	
+  describe 'sshd_config_compression param' do
```

Comment:
I added in the section...let me know if I am missing anything else. Thanks for all the helpful feedback and tips!